packages/gestalt-charts/src/Chart.js

```diff
@@ -1,23 +1,402 @@
-// @flow strict
-import { type Node } from 'react';
-import { Text } from 'gestalt';
+// @flow strict-local
+import { Fragment, type Node, useMemo } from 'react';
+import {
+  BarChart,
+  CartesianGrid,
+  ComposedChart,
+  Legend,
+  LineChart,
+  ResponsiveContainer,
+  Tooltip,
+  XAxis,
+  YAxis,
+} from 'recharts';
+import { Box, Flex, Heading, IconButton, Text, useColorScheme, useDefaultLabel } from 'gestalt';
+import { ChartProvider } from './Chart/ChartContext.js';
+import EmptyBox from './Chart/EmptyBox.js';
+import LegendIcon from './Chart/LegendIcon.js';
+import renderCustomTooltip from './Chart/renderCustomTooltip.js';
+import renderDefaultLegend from './Chart/renderDefaultLegend.js';
+import renderDefaultTooltip from './Chart/renderDefaultTooltip.js';
+import renderElements from './Chart/renderElements.js';
+import renderReferenceAreas from './Chart/renderReferenceAreas.js';
+import usePatterns, { useHexColor } from './Chart/usePatterns.js';
 
-export type Props = {|
-  id?: string,
+type Props = {|
+  accessibilityLabel: string,
+  /**
+   * Prop description.
+   */
+  accessibilitySelected: boolean,
+  /**
+   * Prop description.
+   */
+  biaxial?: 'xAxis' | 'yAxis',
+  /**
+   * Prop description.
+   */
+  data: $ReadOnlyArray<{|
+    name: string | number,
+    [string]: number,
```

Comment:
Any reason were' using keys like this?
```
data = { name:string, values:number[]}
```

Though the format seems okay, it seems like the values represent X axis bits, and not the groups. E.g.
It seems like for the linegraph this can be an issue, because it's up to the consumer to compute the stops the x-axis scale when passing the data.